docs/winshell/INSTALL.md

```diff
@@ -0,0 +1,32 @@
+# OpenKH Documentation - WinShell Extensions
+In order to allow many quality of life improvements to aid in modding Kingdom Hearts games, OpenKH features several WinShell extensions to serve many different purposes.
+These WinShell extensions negate the need to navigate through a tool for the most basic operations you can do with Kingdom Hearts files.
+
+Currently, the extensions you see below are the extensions we have within the project:
+- OpenKh.WinShell.IMDUtilities -> Provides thumbnail generation for IMD files and quick conversion of them to PNG.
+- OpenKh.WinShell.IMZUtilities -> Provides ease of extraction and repack for IMZ files.
+
+# Installation
+All OpenKh.WinShell libraries are powered by SharpShell. So their installation is not standard, but is easy to do none-the-less. 
+To install these extensions, download the SharpShell ServerManager from [this link](https://github.com/dwmkerr/sharpshell/releases).
+
+Afterwards, open the ServerManager (you need Administrator Privileges for this). You will be greeted with this screen:
+
+![](https://i.imgur.com/VpByIAO.png)
```

Comment:
Between the square brackets, always add a brief description of what the image is about.
I also suggest to host those images in the docs rather than in a different domain.

Reply:
First point is understood.
GitHub didn't let me do the second one, but I shall try again.